README.md

```diff
@@ -0,0 +1,107 @@
+
+Check out the example project: [spark-example-drupal8](https://github.com/BluesparkLabs/spark-example-drupal8).
+
+Here are the main steps outlined.
+
+**1. Add Spark as a dependency:**
+
+        $ composer require bluesparklabs/spark
+
+**2. Define a new script in your `composer.json`:**
+
+```javascript
+"scripts": {
+  "spark": "SPARK_WORKDIR=`pwd` robo --ansi --load-from vendor/bluesparklabs/spark"
+}
+```
+
+**3. Add autoload information to your `autoload` field in your `composer.json`:**
+
+```javascript
+"autoload": {
+    "psr-4": {
+        "BluesparkLabs\\Spark\\": "./vendor/bluesparklabs/spark/src/"
+    }
+},
+```
+
+**4. Create a file named `.spark.yml` in your project's root.** This will be your project-specific configuration that Spark will use.
+
+To learn about how to write your configuration, please refer to the [spark-example-drupal8 project's `.spark.yml` file](https://github.com/BluesparkLabs/spark-example-drupal8/blob/master/.spark.yml).
+
+### Recommended `composer.json` bits
+
+See the [spark-example-drupal8 project's `composer.json` file](https://github.com/BluesparkLabs/spark-example-drupal8/blob/master/composer.json).
+
+**1.** Composer by default installs all packages under a directory called `./vendor`. **Use [`composer/installers`](https://packagist.org/packages/composer/installers) to define installation destinations** for Drupal modules, themes etc. Example configuration in `composer.json`:
+
+```javascript
+"extra": {
+    "installer-paths": {
+        "web/core": ["type:drupal-core"],
+        "web/libraries/{$name}": ["type:drupal-library"],
+        "web/modules/contrib/{$name}": ["type:drupal-module"],
+        "web/profiles/contrib/{$name}": ["type:drupal-profile"],
+        "web/themes/contrib/{$name}": ["type:drupal-theme"],
+        "drush/contrib/{$name}": ["type:drupal-drush"]
+    }
+}
+```
+
+**2.** In case your site is Drupal 8 ([Drupal 7 support is coming](https://github.com/BluesparkLabs/spark/issues/15)), **use [drupal-composer/drupal-scaffold](https://packagist.org/packages/drupal-composer/drupal-scaffold) to install and update files that are outside of the `core`** folder and which are not part of the [drupal/core](https://packagist.org/packages/drupal/core) package. This Composer plugin will take care of the files whenever you install or update `drupal/core`, but to run it manually, you can add a script to your `composer.json`:
+
+```javascript
+"scripts": {
+    "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
+},
+```
+
+**3.** Spark has a command, `drupal:files`, to ensure the `files` folder exists with the right permissions, and that there's a `settings.php` file and a `settings.spark.php` which will have Spark-specific configuration, i.e. database connection etc. You may want to add this command to your `scripts` field in your `composer.json`, so that Composer executes it when packages are installed or updated:
```

Comment:
I feel like we need to make it clear somehow that the settings.spark.php is really a Docker-specific configuration.  Spark can be run on the localhost without docker and without settings.spark.php

Reply:
I'm going to work on making sure `settings.spark.php` can support our "hybrid" approach, too, when PHP and the web server runs locally, but the database etc. remain in containers. Until then, I pushed a change to clarify its current role. Does this look good to you?